README.md

```diff
@@ -1,28 +1,39 @@
+
+## What is Juju
 The [Juju Charmed Operator Lifecycle Manager (OLM)](https://juju.is/docs/olm) is a hybrid-cloud application management and orchestration system for installation and day 2 operations. It helps deploy, configure, scale, integrate, maintain, and manage Kubernetes native, container-native and VM-native applications—and the relations between them.
 
 A charmed operator (also known, more simply, as a “charm”) encapsulates a single application and all the code and know-how it takes to operate it, such as how to combine and work with other related applications or how to upgrade it. Charms are programmed to understand a single application, its operations, and its potential to communicate or integrate with other applications. A charm defines and enables the channels by which applications connect.
 
 ### Cloud vendor agnostic
-
 The instructions cover the deployment of FINOS Legend charmed operators on a host PC along with a locally deployed Gitlab instance. However these charms can be deployed on any cloud and should be able to use any Gitlab instance with requisite permissions.
 
 ### Multi-hybrid cloud
-
 The Legend applications stack was deployed as a _bundle_ in the same cloud. [Juju](https://juju.is/) allows, however, for you to deploy each application on a different cloud and then integrate the stack across your estate.
 
 ### Offline installation
-
 We assume your host had a functioning Internet connection. However it is also possible to [deploy charmed operators offline](https://juju.is/docs/olm/working-offline).
 
 ## Installation
-To get started, you can checkout the [local run documention](LOCAL_RUN.md), which will walk you through and explain all the different deployment steps to run a local Legend instance, and will point you to docs for alternative deployments, such as clouds, barebone installations and other.
+This repository hosts the following tutorials to get started with Legend Charm from the ground up:
+- The [local run documention](docs/deploy/local.md) guides you through the entire setup to run Legend in your local workstation, using microk8s; the documentation was tested on Ubuntu and MacOS workstations, but it should work on Windows workstations too
+- The [AWS EKS documentation](docs/deploy/aws-eks.md) explains how to run Legend on a AWS EKS cluster from scratch.
 
-### Related Repositories
+If you'd like to see the cloud of your choice in this list, do not hesitate to [create an issue](https://github.com/finos/legend-integration-juju/issues)
+
+## Versioning
+The Legend Charm bundle is composed by several charms, which are hosted in multiple github repositories (see below).
+
+Bundles and charms can be released through 2 different channels: **stable** and **edge**; the stable bundle will use stable charm versions, and similarly edge bundles will use edge charm versions.
```

Comment:
This is good enough for now @maoo . In the future when we implement the automation for `stable/2021`, `stable/2022` etc we can refine this documentation then.